lib/commands/certificate.js

```diff
@@ -156,8 +151,8 @@ async function installPre122Certificate (driver) {
   // The second one confirms the previous
   await B.delay(1500);
   await clickElement(driver, Button.Install);
-  // Accept sheet alert
-  await clickElement(driver, Sheet.Install);
+  // Accept sheet/alert
+  await clickElement(driver, Button.Install);
```

Comment:
As far as I remember there might be a race condition here when the sheet is still not shown and the previous dialog is visible while clickElement is executed. We have to make sure this locator won't match the `Install` label (the one ion the right top corner) from the previous view.

Reply:
I could revert this to use the class chain but have the root node on iPad be an `XCUIElementTypeAlert` whilst the iPhone remains as an `XCUIElementTypeSheet` or I can verify that there are at least 2 butttons with the text 'Install' before trying to click on the visible one to avoid branching code.
Or would just having a hardcoded sleep suffice?

Reply:
I would try to distinguish the buttons based on the parent container. That is why the class chain locator has been used for `Sheet`. Check which unique parent element the Install button has on iPad and tune the locator to include it, for example ```**/*[`type == 'XCUIElementTypeAlert' OR type == 'XCUIElementTypeSheet'`]/...```